src/Core/Prefab/Globe/BuilderEllipsoidTile.js

```diff
@@ -79,9 +87,37 @@ BuilderEllipsoidTile.prototype.getUV_PM = function getUV_PM(params) {
     return t - params.deltaUV1;
 };
 
+// return common extent to pool the geometries
+// the geometry in common extent is identical to the existing input
+// with a transformation (translation, rotation)
+BuilderEllipsoidTile.prototype.getCommonGeometryExtent = function getCommonGeometryExtentFn(extent) {
```

Comment:
I'd prefer to reserve `getXXX` for simple getter method.

I have more comment about this API, see layer.